File: GooglePay/src/main/java/com/braintreepayments/api/GooglePayIsReadyToPayCallback.java
package com.braintreepayments.api;

/**
 * Callback for receiving result of
 * {@link GooglePayClient#isReadyToPay(android.content.Context, GooglePayIsReadyToPayCallback)} and
 * {@link GooglePayClient#isReadyToPay(android.content.Context, ReadyForGooglePayRequest, GooglePayIsReadyToPayCallback)}.
 */
public interface GooglePayIsReadyToPayCallback {

    /**
     * @param googlePayReadinessResult
     */
    void onGooglePayReadinessResult(GooglePayReadinessResult googlePayReadinessResult);
}

File: GooglePay/src/main/java/com/braintreepayments/api/GooglePayIsReadyToPayCallback.kt
package com.braintreepayments.api

/**
 * Callback for receiving result of [GooglePayClient.isReadyToPay] and
 * [GooglePayClient.isReadyToPay].
 */
fun interface GooglePayIsReadyToPayCallback {

    /**
     * @param googlePayReadinessResult a [GooglePayReadinessResult] signifying if a user is ready to
     * pay.
     */
    fun onGooglePayReadinessResult(googlePayReadinessResult: GooglePayReadinessResult?)
}
